packages/features/bookings/components/event-meta/Duration.tsx

```diff
@@ -9,7 +9,10 @@ import { Badge } from "@calcom/ui";
 import type { PublicEvent } from "../../types";
 
 /** Render X mins as X hours or X hours Y mins instead of in minutes once >= 60 minutes */
-const getDurationFormatted = (mins: number, t: TFunction) => {
+export const getDurationFormatted = (mins: number | null | undefined, t: TFunction) => {
```

Comment:
This won't work if mins is undefined

Reply:
Hi, would you mind being more specific about why it won't work? Sorry if this is relatively basic, I'm still a student :).

Reply:
you were only returning null when mins == null and not checking undefined

Reply:
Hi, I was basing my code on this answer (https://stackoverflow.com/questions/28975896/is-there-a-way-to-check-for-both-null-and-undefined) which much not be correct... Just wanted to bring to your attention that the way you corrected this will not work if the length of an event is 0 minutes (don't know if that's a possibility but that's why I had coded it the way I did instead of the way you did). Cheers